### PR TITLE
Updates to ResponsiveDialogContent

### PR DESCRIPTION
**Pull request by ssancho14**
* Added a message parameter to ResponsiveDialogContent, to simplify the most common use-case.
* Removed SidePaddingExtraPct
* Updated tests.
* Other Minor API changes.

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt [hunk 1]
```diff
@@ -87,28 +88,44 @@ public fun ResponsiveDialogContent(
...
-                            SidePaddingExtraPct(
-                                8.84f,
-                                Modifier.padding(bottom = 8.dp), // 12.dp bellow icon
+                            // Compute the actual padding needed to get and extra 8.84% extra,
+                            // inside of the global 5.2% padding
+                            val extraPadding = 8.84f / (100f - 2f * 5.2f)
+                            Box(
+                                Modifier
+                                    .fillMaxWidth(1f - 2f * extraPadding)
+                                    .padding(bottom = 8.dp), // 12.dp bellow icon
```
Comment (yschimke) [inline compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt:97]:
```suggestion
                                    .padding(bottom = 8.dp), // 12.dp below icon
```

File: sample/src/test/snapshots/images/com.google.android.horologist.screensizes_DialogTest_screenshot[0]_mobvoiticwatchpro5.png
Comment (yschimke) [inline sample/src/test/snapshots/images/com.google.android.horologist.screensizes_DialogTest_screenshot[0]_mobvoiticwatchpro5.png:1]: Do you expect such a big difference in message padding? or just unlucky based on word breaks?

**Review by yschimke — APPROVED** [review]

**Comment by yschimke** [comment]
```
/home/runner/work/horologist/horologist/compose-material/build/metalava/current.txt:5: error: Attempted to change parameter name from body to message in method com.google.android.horologist.compose.material.AlertDialogKt.AlertContent [ParameterNameChange]
```

You'll need to update the api files.

`./gradlew metalavaGenerateSignature `

File: compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt [hunk 2]
```diff
@@ -87,28 +88,44 @@ public fun ResponsiveDialogContent(
                         CompositionLocalProvider(
                             LocalTextStyle provides MaterialTheme.typography.title3,
                         ) {
-                            SidePaddingExtraPct(
-                                8.84f,
-                                Modifier.padding(bottom = 8.dp), // 12.dp bellow icon
+                            // Compute the actual padding needed to get and extra 8.84% extra,
```
Comment (luizgrp) [inline compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt:91]:
```suggestion
                            // Compute the actual padding needed to get an extra 8.84%,
```

File: compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt [hunk 3]
```diff
@@ -87,28 +88,44 @@ public fun ResponsiveDialogContent(
...
-                        }
+                if (icon == null && title == null) {
+                    // Ensure the content is visible when there is nothing above it.
+                    item {
+                        Spacer(Modifier.height(20.dp))
+                    }
+                }
+                message?.let {
+                    item {
+                        // Compute the actual padding needed to get and extra 4.16% extra,
```
Comment (luizgrp) [inline compose-material/src/main/java/com/google/android/horologist/compose/material/ResponsiveDialog.kt:110]:
```suggestion
                        // Compute the actual padding needed to get an extra 4.16%,
```

File: sample/src/test/snapshots/images/com.google.android.horologist.screensizes_DialogTest_screenshot[0]_mobvoiticwatchpro5.png
Comment (ssancho14) [reply sample/src/test/snapshots/images/com.google.android.horologist.screensizes_DialogTest_screenshot[0]_mobvoiticwatchpro5.png:1]: I think the second, maybe "Install" almost fit, but it doesn't so the whole word needs to go down.

**Review by luizgrp — APPROVED** [review]
LGTM

need to update metalava with gradle task: `compose-material:metalavaGenerateSignatureRelease `

**Pull request merged**